templates/ecs-cluster.rb

```diff
@@ -297,7 +297,7 @@
       "service docker stop\n",
       "service docker start\n",
       "start ecs\n",
-      "docker run --name jenkins-docker-slave --privileged=true -d -e PORT=4444 -p 4444:4444 -p 2223:22 -v /data/jenkins-dind/:/var/lib/docker base2/ciinabox-dind-slave\n",
+      "docker run --name jenkins-docker-slave --privileged=true -d -e PORT=4444 -p 4444:4444 -p 2223:22#{(not defined? volatile_jenkins_slave or not volatile_jenkins_slave)? '': ' -v /data/jenkins-dind/:/var/lib/docker '}base2/ciinabox-dind-slave\n",
```

Comment:
@Shuliyey my understanding is that by default this would leave `/var/lib/docker` running on container rather than being mounted from host, right? Wouldn't we want to have this mounted by default? Also in case that `volatile_jenkins_slave` is not defined, this would result in faulty command, with no space between port and container name.

```
$ irb
2.3.0 :001 > "docker run --name jenkins-docker-slave --privileged=true -d -e PORT=4444 -p 4444:4444 -p 2223:22#{(not defined? volatile_jenkins_slave or not volatile_jenkins_slave)? '': ' -v /data/jenkins-dind/:/var/lib/docker '}base2/ciinabox-dind-slave\n"
 => "docker run --name jenkins-docker-slave --privileged=true -d -e PORT=4444 -p 4444:4444 -p 2223:22base2/ciinabox-dind-slave\n" 
```

- there is no space between  container name and port definitions.